site/content/docs/mdx.md

```diff
@@ -0,0 +1,288 @@
+import { Callout } from '../components/TempCallout.jsx'
+
+# MDX
+
+Flowershow parses all of your Markdown files as MDX. This means you not only can write your content using good old Markdown, but also extend it with JSX, JavaScript expressions, and `import` and `export` statements. It's basically Markdown on steroids 😀💪!
+
+Let's see what exactly is MDX and what's so cool about it!
+
+<Callout>
+  A basic familiarity with JSX (React components) and JavaScript might be helpful to understand this chapter, but you can also learn by example and start by tweaking some of our code. Opening this page in your text editor side by side with the rendered version in your browser may also help.
```

Comment:
🚩 Opening this page in your text editor => link `this page` to the source in github (?)